src/sentry/utils/registry.py

```diff
@@ -15,9 +15,10 @@ class NoRegistrationExistsError(ValueError):
 
 
 class Registry(Generic[T]):
-    def __init__(self):
+    def __init__(self, enable_reverse_lookup: bool = True):
```

Comment:
We might not need these changes if we don't actually need the registry, but maybe they're a good addition anyway? Could be split into a separate pr if not required here

Reply:
took it out for now